releasenotes/notes/default-optimization-level-3197ff933dfcfa4f.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
```

Comment:
```suggestion
```
The extra space may mess with the docs formatting 
This release note can be categorized as "upgrade"